book/getting-started/linux-setup.md

```diff
@@ -169,16 +169,41 @@ cd ~/wallaroo-tutorial/wallaroo/machida
 make
 ```
 
+### Compiling on Artful
```

Comment:
Perhaps remove the `make` line at 169, since the instances of `make` below are supposed to replace it?